remme/rest_api/openapi.yml

```diff
@@ -117,52 +117,52 @@ paths:
               - balance
```

Comment:
certificate methods should be as it is. 
but we need to add additional **pub_keys** endpoints (check and get_user_public_keys)